Makefile

```
@@ -65,18 +65,20 @@ install: check-bbr download-v3-binaries
 	@go install $(BUILD_FLAGS_MULTIPLEXER) ./cmd/celestia-appd
 .PHONY: install
 
-## download-v3-binaries: Download the binaries for the latest v3.x.x release.
+## download-v3-binaries: Download the binary for the current platform for v3.
 download-v3-binaries:
-	@echo "--> Downloading embedded binaries for v3"
-	@for pair in \
-		"celestia-app_Darwin_arm64.tar.gz:celestia-app_darwin_v3_arm64.tar.gz" \
-		"celestia-app_Linux_arm64.tar.gz:celestia-app_linux_v3_arm64.tar.gz" \
-		"celestia-app_Darwin_x86_64.tar.gz:celestia-app_darwin_v3_amd64.tar.gz" \
-		"celestia-app_Linux_x86_64.tar.gz:celestia-app_linux_v3_amd64.tar.gz"; do \
-		url=$${pair%%:*}; out=$${pair##*:}; \
-		$(EMBED_BIN); \
-	done
-	@echo "--> Downloaded embedded binaries for v3"
+	@echo "--> Downloading embedded binary for v3"
```

[nit] would be nice if this logged the exact celestia-app version.

```suggestion
	@echo "--> Downloading celestia-app $(CELESTIA_V3_VERSION) binary"
```